src/app/utils/graph-toolkit-lookup.ts

```diff
@@ -8,7 +8,7 @@ export function lookupToolkitUrl(sampleQuery: IQuery) {
     const { requestUrl, search } = parseSampleUrl(sampleQuery.sampleUrl);
     const query = '/' + requestUrl + search;
     for (const templateMapKey in templates) {
-      if (templates.hasOwnProperty(templateMapKey)) {
+      if (Object.prototype.hasOwnProperty.call(templates, templateMapKey)) {
```

Comment:
Why the change?

Reply:
This is because of the eslint update, leaving it as is presents and error. 
Reference this https://eslint.org/docs/rules/no-prototype-builtins